openapi.yaml

```
@@ -5550,8 +5550,18 @@ paths:
                   $ref: '#/components/schemas/Service'
         '401':
           $ref: '#/components/responses/401'
-        '400':
-          $ref: '#/components/responses/400'
+        '409':
+          description: 'Domain conflicts detected.'
+          content:
+            application/json:
+              schema:
+                properties:
+                  message: { type: string, example: 'Domain conflicts detected. Use force_domain_override=true to proceed.' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)